support tegra (orin) build for CTK12.X

billysuh7 opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->
Providing orin build for https://github.com/conda-forge/cuda-feedstock/issues/57
<!--
Please add any other relevant info below:
-->

billysuh7 committed:
add tegra support

billysuh7 committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.14.10.19.06

Other tools:
- conda-build 25.3.2
- rattler-build 0.39.0
- rattler-build-conda-compat 1.4.5

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/18575653474](https://github.com/conda-forge/conda-forge-webservices/actions/runs/18575653474). Examine the logs at this URL for more detail.</sub>

billysuh7 committed:
Bump build number

carterbox commented on .ci_support/linux_64_.yaml:
```diff
@@ -1,21 +1,19 @@
 c_stdlib:
 - sysroot
 c_stdlib_version:
-- '2.17'
```
Why does the minimum glibc version increase from 2.17 to 2.28 for x86 and sbsa?

billysuh7 replied:
Sorry, for no specific reason... I thought 2.17 would be too old. But seeing that we still have 2.17 in the global pinning, I can go back to 2.17 for them. For tegra I do need 2.34, so it will stay. I'll revert for x86 / sbsa.

billysuh7 committed:
Revert to glibc 2.17 for x86, sbsa since that was the previous version for them

billysuh7 committed:
MNT: Re-rendered with conda-smithy 3.52.3 and conda-forge-pinning 2025.10.15.11.56.55

Other tools:
- conda-build 25.3.2
- rattler-build 0.39.0
- rattler-build-conda-compat 1.4.5

billysuh7 commented:
Thanks @carterbox!
